src/app/lab/lab.component.ts

```diff
@@ -89,9 +93,13 @@ export class LabComponent implements OnInit, OnDestroy {
             return;
         }
         this.removeChart();
-        this.indicator = indicator;
-        const chart = new Chart({indicator: indicator,
-                                 unit: indicator.default_units});
-        this.project.project_data.charts = [chart];
+        // Trigger lifecycle evaluation to truly destroy the chart child component
```

Comment:
This comment should probably have a bit more description of _why_ we want to completely destroy this component when we select a different indicator, so the context is apparent for the next person that looks at this.